docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,322 @@
+
+```python
+from qiskit_ibm_runtime import Options
+
+options = Options(optimization_level=1, environment={"log_level": "INFO"})
+```
+
+The `Options` class supports auto-complete. Once you create an instance of the `Options` class, you can use auto-complete to see what options are available. If you choose one of the categories, you can use auto-complete again to see what options are available under that category.
+
+```python
+from qiskit_ibm_runtime import Options
+
+options = Options()
+options.resilience_level = 1
+options.execution.shots = 2048
+```
+<span id="pass-options"></span>
+## Set primitive options
+
+You can set options when initializing the primitive, after initializing the primitive, or in the `run()` method. See the [precidence rules](runtime-options-overview#options-precedence) section to understand what happens when the same option is specified in multiple places.
+
+### Primitive initialization
+
+You can pass in an instance of the options class or a dictionary when initializing a primitive.
+
+#### Options class
+
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in an instance of the options class. Those options will then be applied when you use `run()` to perform the calculation.  Specify the options in this format:  `primitive.options.option.sub-option.sub-sub-option = choice`.  For example: `estimator.options.dynamical_decoupling.enable = True`
+
+Example:
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+    `SamplerV2` and `EstimatorV2` have separate options classes ([`EstimatorOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions) and [`SamplerOptions`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.SamplerOptions)) that do not need to be instantiated, but they can be if you want to pass them in during primitive initialization.
+
+```python
+from qiskit_ibm_runtime.options import EstimatorOptions
+
+options =  EstimatorOptions(resilience_level=2, resilience={"zne_mitigation": True, "zne": {"noise_factors": [1, 3, 5]}})
+
+# or...
+options =  EstimatorOptions()
+estimator.options.resilience_level = 2
+options.resilience.zne_mitigation = True
+options.resilience.zne.noise_factors = [1, 3, 5]
+
+estimator = Estimator(mode=backend, options=options)
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+```python
+from qiskit_ibm_runtime import Estimator, Options
+
+options = Options(resilience_level = 2)
+
+# or...
+
+options = Options()
+options.resilience_level = 2
+
+estimator = Estimator(session=backend, options=options)
+```
+  </TabItem>
+</Tabs>
+
+#### Dictionary
+
+You can specify options as a dictionary when initializing the primitive.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+ ```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
```

Comment:
nit - it seems pretty random what import / initialization is done. For example, this code example has 

`from qiskit_ibm_runtime import QiskitRuntimeService` and `service = QiskitRuntimeService()`, but the previous one doesn't.

Reply:
True :)